internal/apiresourceset/tektonapiresourceset.go

```diff
@@ -0,0 +1,469 @@
+/*
```

Comment:
We should use the apiresource interface, it helps in targetting resources to a particular cluster version and stuff.

Reply:
Lets do it as a separate issue, so that we can get early feedback of it.